apps/website/src/pages/index.astro

```diff
@@ -71,6 +72,13 @@ const currentPage = new URL(Astro.request.url).pathname;
           <h3>ComboBox</h3>
           <p>Allows typing a value to filter and choose from the options in dropdown list.</p>
         </div>
+        <div class='demo-wrapper'>
+          <div class='demo-box demo-box-muted'>
+            <a class='discover-more' href='/docs/components'
+              >Discover more components <MoreIcon />
```

Comment:
Arrow makes more sense compared to carets.

![](https://user-images.githubusercontent.com/9084735/206535183-a6b2aeba-478d-45f2-9a5f-ce6ffb7ebbc2.png)

Reply:
Added arrow